eo-parser/src/main/java/org/eolang/parser/Objects.java

```diff
@@ -69,6 +70,13 @@ interface Objects extends Iterable<Directive> {
      */
     void xprop(String key, Object xpath);
 
+    /**
+     * Add property depends on given condition.
+     * @param condition Condition
+     * @param key Key
+     */
+    void oprop(boolean condition, String key);
```

Comment:
@volodya-lombrozo first "o" stands for "optional". Property that will be added depends on giving condition

Reply:
@maxonfjvipon Why do we need such a method, if in case of `condition==false` we can just avoid calling this method?

Reply:
@volodya-lombrozo this method is used 8 times in `XeListener`. We either call `oprop` 8 times or replace it with `if (this.versioned) { this.objects.prop(key); }` 8 times

Reply:
@maxonfjvipon I don't know how it happened that we have `xprop` method, but both of them `xprop` and `oprop`:
1. Hard readable.
2. Might produce some [issues](https://www.yegor256.com/2023/08/01/method-overloading.html).

Reply:
@volodya-lombrozo there's nothing about overloading
1. `prop` - just adds property, uses `attr` from Xemlby
2. `xprop` - changes property by given xpath, uses `xattr` from Xembly. It was added here: #2474 
3. `oprop` - addes property if given condition is TRUE. Actually is a temporary method. When object versioning is implemented it will be removed and all its occurrences will be replaced with `prop`